src/main/java/catan/controllers/PlayController.java

```diff
@@ -86,6 +87,16 @@ public void throwDice(@RequestParam(value = "token", required = false) String to
         playService.processAction(GameUserActionCode.THROW_DICE, user, gameId);
     }
 
+    @RequestMapping(value = "buy/card",
+            method = RequestMethod.POST,
+            produces = MediaType.APPLICATION_JSON_VALUE)
+    public Map<String, String> buyCard(@RequestParam(value = "token", required = false) String token,
```

Comment:
I don't like that controller returns Map instead of DTO, we should create separate Java class for each controller's response